U/Uno/build_tarballs.jl

```diff
@@ -77,12 +72,13 @@ platforms = supported_platforms()
 filter!(p -> !(Sys.isfreebsd(p) && arch(p) == "aarch64"), platforms)
 filter!(p -> arch(p) != "riscv64", platforms)
 platforms = expand_cxxstring_abis(platforms)
+platforms = expand_gfortran_versions(platforms)
```

Comment:
@cvanaret 
https://github.com/JuliaPackaging/Yggdrasil/blob/master/G/GALAHAD/build_tarballs.jl#L135-L136

Reply:
Why dropping libgfortran4? That's working, isn't it?

Reply:
We can keep it but who is still using it?  Uno also requires a recent GCC (10.x), I am scared that it was a pain to maintain for GCC 7, required for `libgfortran4`.